snapcraft/repo.py

```diff
@@ -298,6 +308,14 @@ def _fix_filemode(path):
         os.chmod(path, mode & 0o1777)
 
 
+def _fix_shebangs(path):
```

Comment:
It wouldn't hurt to comment why the shebangs have to be fixed. Just like on fix_symlinks above.